aws-greengrass-testing-platform/aws-greengrass-testing-platform-pillbox/pom.xml

```diff
@@ -124,7 +124,7 @@
         <dependency>
             <groupId>com.google.guava</groupId>
             <artifactId>guava</artifactId>
-            <version>31.1-jre</version>
+            <version>23.0</version>
```

Comment:
Why is this updated?

Reply:
+1

Reply:
resolved

Reply:
Mmm I don't know why this shows up as changed but we have to make sure this has the previous value. We don't want to update this dependencies if it is not needed.